apiary.apib

```diff
@@ -3141,6 +3142,52 @@ Update a draft invoice. Booked invoices cannot be updated.
 
 + Response 204
 
+### invoices.send [POST /invoices.send]
+
+Send an invoice via e-mail.
+
++ Request (application/json)
+    + Attributes (object)
+        + id: `d885e5d5-bacb-4607-bde9-abc4a04a901b` (string, required)
+        + content (object, required)
+             + subject: `Invoice` (string, required)
+             + body: `Please find your invoice attached to this mail` (string, required)
+             + template_id: `045cd6a9-7527-09c9-aa16-fd5ac7953e71` (string, optional, nullable)
+        + recipients (object, required)
+            + to (array, required)
+                + (object)
+                    + customer (object, optional, nullable)
+                        + type (enum, required)
+                            + Members
+                                + company
+                                + contact
+                        + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
+                    + email: `info@teamleader.eu` (string, required)
+            + cc (array, optional)
+                + (object)
+                    + customer (object, optional, nullable)
+                        + type (enum, required)
+                            + Members
+                                + company
+                                + contact
+                        + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
+                    + email: `info@teamleader.eu` (string, required)
+           + bcc (array, optional)
+                + (object)
+                   + customer (object, optional, nullable)
+                       + type (enum, required)
+                           + Members
+                               + company
+                               + contact
+                       + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
+                   + email: `info@teamleader.eu` (string, required)
+       + attachments: `e76d381a-f7ea-04d1-9a32-14aab27bc1e2` (array[string], optional) - an array of file ids
+       + allow_reminders: true (boolean, optional) - If set to false, the endpoint will return a 400 response if invoice was already sent
+           + Default: true
+       + attach_invoice_pdf: true (boolean, optional) - If set to true, the invoice's pdf will be included as an attachment to the email
```

Comment:
Q: We do not require this property on `quotations.send` why do we require this here?

Reply:
Not sure I understand, it's not required but optional.
AFAIK the pdf may not be attached if f.e. it will be instead made available in invoicecloud. 

> Seems like we have a behavioural difference we'll need to adjust first.
Yes, but invoices and quotations are not the same so having slightly different behaviour is to be expected, no?

Reply:
> Not sure I understand, it's not required but optional.

I meant having the property available.